### PR TITLE
DOC add sensor generalization tutorial

### PR DESCRIPTION
**Pull request by bruAristimunha**
## Summary
- Add a tutorial showing generalization across sensors with time samples as features.
- Position sensor generalization alongside temporal generalization.
- Use the visual_92_categories dataset and remove local reference-implementation artifacts.

## Verification
- Tutorial script executed successfully.
- Ruff check passed.
- Patterned documentation build completed; only the environment's missing optipng warning remains.

### CODE REVIEW

**Comment by larsoner** [comment]
When CircleCI comes back can you check the computation memory and time it required? It should show on the redirector landing page (it links to the "computation time" page)

File: mne/decoding/search_light.py [hunk 1]
```diff
@@ -54,13 +55,15 @@ def __init__(
         position=0,
         allow_2d=False,
         verbose=None,
+        axis=-1,
```
Comment (larsoner) [inline mne/decoding/search_light.py:57]: If we're going to add this, we'll want some tests to accompany as well. Please try to keep them compact. I don't think they need to be as comprehensive as existing ones. But I do wonder the extent to which other existing `SlidingEstimator` methods work when `axis != -1`

File: tutorials/machine-learning/60_gal_decoding.py [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+from sklearn.model_selection import StratifiedKFold
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+
+import mne
+from mne.channels import make_standard_montage
+from mne.decoding import GeneralizingEstimator, SlidingEstimator, cross_val_multiscore
+from mne.io import read_raw_eeglab
+
+data_dir = mne.datasets.default_path() / "ERP-CORE-N170" / "sub-001" / "eeg"
```
Comment (larsoner) [inline tutorials/machine-learning/60_gal_decoding.py:74]: Since `mne.datasets.erp_core` is not enough... we still want to decouple the download from the example running. This enables better output in the example, and fails early if a download fails because we can front-load it in CircleCI. I wonder if what we want is a `mne.datasets.erp_core.<something>` that can be called here with just filenames, like
```
mne.datasets.erp_core.fetch_file("sub-001/eeg/sub-001_task-N170_eeg.fdt")
```
and we have a registry of just the three files below, and it uses `openneuro-py` to download if not already on disk. Then this file gets much simpler, and in `_download_all_example_data` and in our CircleCI download triage `tools/circleci_download.sh` -- all three just have the `fetch_file` 3 lines in them.

**Comment by larsoner** [comment]
Nice! I looked at the code and tutorial and it looks reasonable to me. Thanks @bruAristimunha !

**Comment by larsoner** [comment]
Oh actually @ASantosMayo also contributed as well, thanks!

@bruAristimunha thinking more this probably warrants a changelog entry, can you add one?

File: doc/changes/dev/14211.newfeature.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Added :ref:`tut-gal-decoding`, a tutorial on Generalization Across Location
+(GAL) and Time-GAL decoding of EEG data, by `Bruno Aristimunha`_ and
+`Alejandro Santos Mayo`_.
```
Comment (larsoner) [inline doc/changes/dev/14211.newfeature.rst]:
```suggestion
:newcontrib:`Alejandro Santos Mayo`.
```

**Review by larsoner — APPROVED** [review]
Thanks @bruAristimunha @ASantosMayo !
